test/lib/ansible_test/_data/injector/virtualenv.sh

```diff
@@ -1,8 +1,17 @@
 #!/usr/bin/env bash
 # Create and activate a fresh virtual environment with `source virtualenv.sh`.
 
+# Privilege venv if it is available but fallback to virtualenv
+# https://github.com/ansible/ansible/issues/72738
+"${ANSIBLE_TEST_PYTHON_INTERPRETER}" -m venv &> /dev/null
```

Comment:
This test is unsafe on Debian based distributions, which will report that `venv` is present when it is not.

Reply:
On a fresh `ubuntu:20.04` container with `python3` installed, but not `python3-venv` we'll see this:

```
# python3 -m venv
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear] [--upgrade] [--without-pip] [--prompt PROMPT]
            ENV_DIR [ENV_DIR ...]
venv: error: the following arguments are required: ENV_DIR
```

Which fails as expected and reports a return code of 2.

However, creating a venv fails:

```
# python3 -m venv /tmp/venv
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt-get install python3-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: ['/tmp/venv/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']
```

Reply:
Ew. I'll figure something out.